demos/demos.http

```diff
@@ -8,6 +8,10 @@ GET {{base_url}}/readyz
 
 ###
 POST {{base_url}}/cdevents
+headers:
+#  ce-spec-version: 1.0
+#  ce-id: 1234
+#  ce-source: /mycontext
```

Comment:
```suggestion
ce-spec-version: 1.0
ce-id: 1234
ce-source: /mycontext
```

you can look at https://httpyac.github.io/guide/request.html if want to know more about the HTTP syntax (it's used by tool like httpyac, restclient, some idea plugin iirc,...)